README.md

```diff
@@ -109,24 +109,29 @@ If you are a library author looking to build OpenTelemetry into your library, pl
 
 ## Supported Runtimes
 
-| Platform Version    | Supported                                       |
-| ------------------- | ----------------------------------------------- |
-| Node.JS `v18`       | ✅                                               |
-| Node.JS `v16`       | ✅                                               |
-| Node.JS `v14`       | ✅                                               |
-| Older Node Versions | See [Node Support](#node-support)               |
-| Web Browsers        | ✅ See [Browser Support](#browser-support) below |
+| Platform Version    | Supported                                     |
+|---------------------|-----------------------------------------------|
+| Node.JS `v18`       | ✅                                             |
+| Node.JS `v16`       | ✅                                             |
+| Node.JS `v14`       | ✅                                             |
+| Older Node Versions | See [Node Support](#node-support)             |
+| Web Browsers        | See [Browser Support](#browser-support) below |
 
 ### Node Support
 
 Only Node.js Active or Maintenance LTS versions are supported.
 Previous versions of node *may* work, but they are not tested by OpenTelemetry and they are not guaranteed to work.
-Please note that versions of Node.JS v8 prior to `v8.12.0` will NOT work, because OpenTelemetry Node depends on the `perf_hooks` module introduced in `v8.5.0` and `performance.timeOrigin` that is set correctly starting in `v8.12.0`.
+Please note that versions of Node.JS v8 prior to `v8.12.0` will NOT work, because OpenTelemetry Node depends on the
+`perf_hooks` module introduced in `v8.5.0` and `performance.timeOrigin` that is set correctly starting in `v8.12.0`.
 
 ### Browser Support
 
-Automated browser tests are run in the latest version of Headless Chrome.
-There is currently no list of officially supported browsers, but OpenTelemetry is developed using standard web technologies with wide support and should work in currently supported versions of major browsers.
+> [!IMPORTANT]
```

Comment:
I'm curious, where is `[!IMPORTANT]` defined?

Reply:
It's part of GitHub Flavored Markdown, see https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#alerts :slightly_smiling_face: